src/core/qgsdataitem.h

```diff
@@ -340,6 +350,7 @@ class CORE_EXPORT QgsLayerItem : public QgsDataItem
     static const QIcon &iconRaster();
     static const QIcon &iconDefault();
 
+    /** returns the layer name */
```

Comment:
`@return` will properly let doxygen pickup the value as description of the return value.